### PR TITLE
Changed the Text in the Edit Image Activity to English

### PR DESCRIPTION
**Pull request by Subhankar29**
Fixes issue #480 

Changes: The texts in the EditImageActivity was written in Chinese. Changed it to English
Screenshots for the change: 
screenshot_2017-05-25-18-20-16-293

### CODE REVIEW

**Comment by Subhankar29** [comment]
@pa1pal @vinaysajjanapu @mohitmanuja please review the PR.

File: imageeditlibrary/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
+        <item>fleeting time</item>
+        <item>Light</item>
     </string-array>
 
-    <string name="more">更多</string>
-    <string name="exit_without_save">编辑的图片还未保存,确认退出吗？</string>
-    <string name="confirm">确认</string>
-    <string name="cancel">取消</string>
+    <string name="more">More</string>
+    <string name="exit_without_save">Exit Withou Saving？</string>
```
Comment (mohitmanuja) [inline imageeditlibrary/src/main/res/values-zh/strings.xml]: Exit Without Saving?

Comment (Subhankar29) [inline imageeditlibrary/src/main/res/values-zh/strings.xml]: Ok, missed that will change it.

**Comment by mohitmanuja** [comment]
@Subhankar29 Try to remove all Chinese content i.e comments, introduction or convert it in english.

**Comment by Subhankar29** [comment]
As it is a tedious process, I will do it in subsequent PRs.

**Pull request merged**